aten/src/ATen/core/interned_strings.h

```diff
@@ -1,9 +1,9 @@
 #pragma once
-#include <vector>
+#include <algorithm>
```

Comment:
Can you revert the formatting changes here in this PR or stack them ?

Reply:
sure, reverted the formatting part.